### DISCUSSION TITLE
Prevent icon from being selected on custom link block

### DISCUSSION THREAD

**Discussion by YannRostingt** [GrapesJS/grapesjs#4491] [Q&A]
Hi eveyone, 

I created a block for social network redirects with svgs icons

![image](https://user-images.githubusercontent.com/47507450/182108599-05a601d2-4fb0-4d7c-b73f-73fdb4885e17.png)

I would like to know how to make sure that the svgs are not selectable in order to facilitate the use of this block

![image](https://user-images.githubusercontent.com/47507450/182109498-f5f68b87-493e-4eb1-a460-1e72632eb76b.png)

![image](https://user-images.githubusercontent.com/47507450/182109517-f40fe7c5-c3d4-42e2-a755-69ad8848567f.png)

I remain attentive to your advice thank you all

**Comment by YannRostingt** [comment]
Nevermind i found a way :)

**Comment by ronaldohoch** [comment]
Hello!

Here, we use these html attributes in link images: `data-gjs-editable="false" data-gjs-selectable="false"`.
